src/main/java/org/folio/rest/impl/PatronServicesResourceImpl.java

```diff
@@ -142,17 +148,22 @@ public void getPatronAccountById(String id, boolean includeLoans,
             return CompletableFuture.allOf(cf1, cf2, cf3)
                 .thenApply(result -> account);
           } catch (Exception e) {
+            logger.warn("getPatronAccountById:: Exception in first thenCompose block while fetching PatronAccount ", e);
             throw new CompletionException(e);
           }
         })
         .thenAccept(account -> {
           asyncResultHandler.handle(succeededFuture(GetPatronAccountByIdResponse.respond200WithApplicationJson(account)));
         })
         .exceptionally(throwable -> {
+          logger.warn("getPatronAccountById:: Exception in exceptionally block for fetching PatronAccount " +
```

Comment:
Think `errror` would be more appropriate

Reply:
fixed=> https://github.com/folio-org/mod-patron/pull/154/commits/90f5d48d8337dc640f9cd7a694bb18d7962bdc7e